src/ensembl/src/shared/badged-button/BadgedButton.tsx

```diff
@@ -11,12 +11,21 @@ type Props = {
 const BadgedButton = (props: Props) => {
   const className = classNames(defaultStyles.badgeDefault, props.className);
 
+  let badgeContent = props.badgeContent;
+
+  if (typeof badgeContent === 'number') {
+    if (badgeContent > 99) {
```

Comment:
Can this condition go inside the previous one (`typeof badgeContent === 'number' && badgeContent > 99`) to avoid the nested if-statement?

Reply:
That's done.